### ISSUE TITLE
status of this project

### ISSUE THREAD

**Issue by iguannago** [Pragmatists/JUnitParams#169]
Hi, 

Firstly, congrats for this great work. We have been using it for quite a long time and we haven't had issues. Now, we would like to use it in another project but there are some concerns about the support of this. It looks like there has been no activity in the last months. 

Could anyone, please, give us some update on this regard? 

Many thanks, 
David

**Comment by ghost** [comment]
Agreed, please confirm that this project is still actively maintained via return comment.
